### PR TITLE
React Goods Selector was done

### PR DESCRIPTION
**Pull request by vladgusakovskiy**
[DEMO LINK](https://vladgusakovskiy.github.io/react_goods-selector/)

### CODE REVIEW

**Review by Andrii-Dunets — APPROVED** [review]
Good job)

File: src/App.tsx [hunk 1]
```diff
@@ -15,71 +16,90 @@ export const goods = [
...
+  };
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  clearButton = () => {
```
Comment (denys-danyliuk) [inline src/App.tsx:28]: now it seems like you naming a button, not a method, try to change it

File: src/App.tsx [hunk 2]
```diff
@@ -15,71 +16,90 @@ export const goods = [
...
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  addButton = (event: React.MouseEvent<HTMLButtonElement>) => (
```
Comment (denys-danyliuk) [inline src/App.tsx:32]: same here

**Review by denys-danyliuk — CHANGES_REQUESTED** [review]
GJ!
Try to find better names for methods

**Review by denys-danyliuk — APPROVED** [review]
GJ!

**Review by denys-danyliuk — APPROVED** [review]

**Pull request closed without merging**